hopp/simulation/technologies/battery/battery_stateless.py

```diff
@@ -64,6 +64,7 @@ class BatteryStatelessConfig(BaseClass):
     maximum_SOC: float = field(default=90, validator=range_val(0, 100))
     initial_SOC: float = field(default=10, validator=range_val(0, 100))
     fin_model: Union[dict, CustomFinancialModel] = field(default=None)
```

Comment:
Let's update this to 
```
fin_model: Optional[Union[str, dict, FinancialModelType]] = field(default=None)
```

Reply:
Done. I also did the same for `battery.py`